src/Proto.Cluster/Partition/PartitionManager.cs

```diff
@@ -49,26 +50,24 @@ public void Setup()
                 var partitionActorProps = Props
                     .FromProducer(() => new PartitionIdentityActor(_cluster))
                     .WithGuardianSupervisorStrategy(Supervision.AlwaysRestartStrategy);
-                _partitionActor = _context.SpawnNamed(partitionActorProps, PartitionIdentityActorName);
+                _partitionIdentityActor = _context.SpawnNamed(partitionActorProps, PartitionIdentityActorName);
 
                 var partitionActivatorProps =
                     Props.FromProducer(() => new PartitionPlacementActor(_cluster));
-                _partitionActivator = _context.SpawnNamed(partitionActivatorProps, PartitionPlacementActorName);
+                _partitionPlacementActor = _context.SpawnNamed(partitionActivatorProps, PartitionPlacementActorName);
 
                 //synchronous subscribe to keep accurate
 
                 var eventId = 0ul;
                 //make sure selector is updated first
                 _system.EventStream.Subscribe<ClusterTopology>(e => {
-                        if (e.EventId > eventId)
-                        {
-                            eventId = e.EventId;
-                            _cluster.MemberList.BroadcastEvent(e);
```

Comment:
Are we sure we can completely skip broadcasting the topology? It will then all need to go via the cluster-provider directly, hopefully simultaneously

Reply:
The problem with broadcasting when we do not have an event index, but just a hash. we don't know which event is the most recent.
e.g one node may get the actual latest event from the cluster provider, and then another lagging node may broadcast a previous topology event, fooling the node with correct data, that there is a new topology.

the DB driven IdentityLookups does not use this kind of broadcast either.